core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1809,15 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the duration between logging uncaught exceptions.
+     * @param exceptionReportingInterval - the duration between logging uncaught exceptions.
+     */
+    public ServerBuilder exceptionReportingInterval(Duration exceptionReportingInterval) {
```

Comment:
Could we also add a shortcut method that accept seconds? `exceptionReportingIntervalSeconds(long ...Seconds)`

Reply:
Great idea!